presidio-anonymizer/presidio_anonymizer/anonymizer_engine.py

```diff
@@ -78,9 +79,22 @@ def anonymize(
             analyzer_results
         )
 
+        # This list merges adjacent entities of the same type
```

Comment:
Can we move this logic into a function? to make the anonymization pipeline more readable?

Reply:
I have moved the logic to a function named `_merge_entities_with_whitespace_between`